CHANGELOG.md

```diff
@@ -1,3 +1,70 @@
+## v1.5.0 (18/08/2020)
+
+**olp-sdk-core**
+
+- Added the new `@here/olp-sdk-core` module v1.0.0 and moved common components to it.
+
+**olp-sdk-authentication**
+
+- Added `@here/olp-sdk-core` as a dependency.
+- Updated `olp-sdk-fetch` to version 1.5.0.
+- Deprecated the `lib.version.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `./lib/HttpError.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+
+**olp-sdk-dataservice-api**
+
+- Added the `checkBlobExistsStatus` function to `blob-api.ts`. It is a fork of the `checkBlobExists` function.
+- Deprecated the `checkBlobExists` function in `blob-api.ts`. It will be removed by 02.2021. Use the `checkBlobExistsStatus` function instead.
+- Added the `doCompleteMultipartUpload` function to `blob-api.ts`. It is a fork of the `completeMultipartUpload` function.
+- Deprecated the `completeMultipartUpload` function in `blob-api.ts`. It will be removed by 02.2021. Use the `doCompleteMultipartUpload` function instead.
+- Added the `putData` function to `blob-api.ts`. It is a fork of the `putBlob` function.
+- Deprecated the `putBlob` function in `blob-api.ts`. It will be removed by 02.2021. Use the `putData` function instead.
+- Added the `doUploadPart` function to `blob-api.ts`. It is a fork of the `uploadPart` function.
+- Deprecated the `uploadPart` function in `blob-api.ts`. It will be removed by 02.2021. Use the `doUploadPart` function instead.
+- Deprecated the `./lib/HttpError.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead..
+
+**olp-sdk-fetch**
+
+- Updated dependencies.
+
+**olp-sdk-dataservice-read**
+
+- Added `@here/olp-sdk-core` as a dependency.
+- Deprecated the `lib.version.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/DataStoreDownloadManager.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/DataStoreRequestBuilder.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/DownloadManager.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/HRN.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/ApiCacheRepository.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
+- Deprecated the `lib/KeyValueCache.ts` file. It will be removed by 02.2021. Use the same the from `@here/olp-sdk-core` package instead.
```

Comment:
```suggestion
- Deprecated the `lib/KeyValueCache.ts` file. It will be removed by 02.2021. Use the same the from the `@here/olp-sdk-core` package instead.
```